src/commands/run-tests.yml

```diff
@@ -40,6 +40,11 @@ parameters:
       parameter requires a Parallel Computing Toolbox license.
     type: boolean
     default: false
+  split-type:
+    description: >
+      Option to use CircleCI Test Split feature, specified as 'timings', 'filename' or 'filesize'.
```

Comment:
I think we need to use the same terminology as in CircleCI documentation. Instead of "CircleCI Test Split feature"  CircleCI uses "CircleCI test splitting" in their documentation: https://circleci.com/docs/parallelism-faster-jobs/#:~:text=CircleCI%20test%20splitting

The information in this description is minimal. For example, it doesn't say anything about the `parallelism` key. At a minimum, we should link to this page: https://circleci.com/docs/parallelism-faster-jobs/. Alternatively, the description can be expanded to include some important details.

Also, it might be good to clarify how one should use the new parameter alongside `use-parallel`.